tagstudio/src/qt/widgets/thumb_renderer.py

```diff
@@ -204,6 +204,40 @@ def render(
                 # 	img_buf = io.BytesIO()
                 # 	plt.savefig(img_buf, format='png')
                 # 	image = Image.open(img_buf)
+
+                # Blender ===========================================================
+                elif _filepath.suffix.lower() == '.blend':
+                    from src.qt.helpers import blender_thumbnailer
```

Comment:
I think this should be moved to the top of the file to the other imports.

Reply:
And i think the import could just import the main out of the blender_thumbnailer so it would be 
from src.qt.helpers.blender_thumbnailer import main

Reply:
Done!